src/state/machines/turn.ts

```diff
@@ -332,33 +367,18 @@ export const TurnMachine = setup({
         // TODO: Figure out what to do when extinction deck is empty
         if (extinctionTiles.length === 0) return;
 
+        draft.extinctMarket.deck = without(context.extinctMarket.deck, ...extinctionTiles);
+        draft.extinctMarket.table = concat(context.extinctMarket.table, extinctionTiles);
+
         draft.stage = {
           eventType: "massExtinction",
-          cause: disasterCards,
-          effect: extinctionTiles,
+          cause: disasterCards.map((disasterCard) => disasterCard.uid),
+          effect: extinctionTiles.map((extinctionTile) => extinctionTile.uid),
```

Comment:
```suggestion
          cause: map(disasterCards,'uid'),
          effect: map(extinctionTiles,'uid'),,
```

also you need to update lodash import:

import { map, compact, concat, countBy, entries, find, intersection, reject, without } from "lodash-es";